docs/commands/gef-remote.md

```diff
@@ -77,14 +78,14 @@ for `gef` will not work either, as `gef` won't be able to fetch the content of
 the remote procfs.
 
 To circumvent this and still enjoy `gef` features with QEMU-user, a simple stub
-can be artificially added, with the option `-q` option of `gef-remote`. Note
-that you need to set the architecture properly first:
+can be artificially added, with the option `--qemu-mode` option of `gef-remote`.
+Note that you need to set the architecture properly first:
```

Comment:
For clarity let's say "set the architecture to match the target binary" or something.

Reply:
"of the inferior" maybe.

Reply:
Good call. To me "target binary" sounds better than "inferior binary"

Reply:
I agree, but I'm willing to let hugsy make the call.